habit-backend/users/admin.py

```diff
@@ -0,0 +1,40 @@
+from .forms import UserAdminCreationForm, UserAdminChangeForm
+from django.contrib import admin
+from django.contrib.auth.models import Group
+from django.contrib.auth.admin import UserAdmin as BaseUserAdmin
+from django.contrib.auth.forms import ReadOnlyPasswordHashField
+
+from users.models import User
+
+class UserAdmin(BaseUserAdmin):
+    # The forms to add and change user instances
+    form = UserAdminChangeForm
+    add_form = UserAdminCreationForm
+
+    # The fields to be used in displaying the User model.
+    # These override the definitions on the base UserAdmin
+    # that reference specific fields on auth.User.
+    list_display = ('email', 'is_superuser')
+    list_filter = ('is_superuser',)
+    fieldsets = (
+        (None, {'fields': ('email', 'password')}),
+        ('Personal info', {'fields': ('first_name','last_name')}),
+        ('Permissions', {'fields': ('is_superuser',)}),
+    )
+    # add_fieldsets is not a standard ModelAdmin attribute. UserAdmin
+    # overrides get_fieldsets to use this attribute when creating a user.
+    add_fieldsets = (
+        (None, {
+            'classes': ('wide',),
+            'fields': ('email', 'password1', 'password2')}
+        ),
+    )
+    search_fields = ('email',)
+    ordering = ('email',)
```

Comment:
Let's order by `-ID` (if we can do the numeric ID thing) OR `-created_date` ?

Reply:
Will change the ordering to id.